pkg/http/api.go

```diff
@@ -249,6 +250,17 @@ func (a *api) constructMetadataEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructHealthzEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{http.Get},
+			Route:   "healthz",
```

Comment:
Why "healthz"? Is "healthz" a convention?

Reply:
> Why "healthz"? Is "healthz" a convention?

Yes - it seems like.

Reply:
I used to use `status` endpoint for readiness and liveness before. but k8s doc uses healthz in doc : https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request